var/spack/repos/builtin/packages/py-pydv/package.py

```diff
@@ -7,13 +7,22 @@
 
 
 class PyPydv(PythonPackage):
-    """PDV is a 1D graphics and data analysis tool, heavily based on the
-    ULTRA plotting tool"""
+    """PyDV is a 1D graphics and data analysis tool, heavily based on the ULTRA plotting tool."""
 
     homepage = "https://github.com/griffin28/PyDV"
-    url = "https://github.com/griffin28/PyDV/archive/pydv-2.4.2.tar.gz"
+    # url = "https://github.com/griffin28/PyDV/archive/pydv-3.1.15.tar.gz"
+    pypi = "llnl-pydv/llnl_pydv-3.1.15.tar.gz"
+    git = "https://github.com/griffin28/PyDV"
+    tags = ["pydv-3.1.15"]
 
-    version("2.4.2", sha256="46bda76e27e85beaad446455d0cc279388d455f05912a8ff8e4fb66de983992c")
```

Comment:
We don't normally delete old versions from packages unless there is some issue with them.